index.html

```diff
@@ -54,7 +54,174 @@ <h1>I’m Godfrey
+      <ul class="conduct">
+        <li class="conduct_head">CANOPY</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_para">Back End Dev</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_year">2015</li>
+      </ul>
+      <p class="work_paragraph">A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+      <ul class="work_languages">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+        <button class="btn">See Project</button>
+    </div>
+
+</article>
+    </div>
+</section>
+<section class="about">
+<article class="about_head">
+  <h2>About
+    Myself</h2>
+  <p>Hello I’m a software developer! 
+    I can help you build a product , 
+    feature or website Look through 
+    some of my work and experience! 
+    If you like what you see and have 
+    a project you need 
+    coded, don’t hestiate to contact me.</p>
+  <a href="">LET’S CONNECT</a>
+  <ul>
+    <li class="item1"><a href=""><img src="./image/Vectorgithub.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Linkedin iconlinkedin.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectormedium.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectortwitter (1).svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vector (3).svg" alt=""></a></li>
+  </ul>
+  <button class="btn">Get my resume</button>
+</article>
+<article class="about_language">
+<div class="item">
+  <h3>Languages</h3>
+  <img src="image/down.svg" alt="">
+</div>
+<ul class="about_lang">
+   <li>
+     <img src="image/javascript.svg" alt="">
+     <p>javascript</p>
+   </li>
+   <li>
+    <img src="image/html (2).svg" alt="">
+    <p>html</p>
+   </li>
+   <li>
+    <img src="image/css.svg" alt="">
+    <p>css</p>
+   </li>
+</ul>
+<li class="item">
+  <h3>Framework</h3>
+  <img src="image/left2.svg" alt="">
+
+</li>
+<hr>
+
+<li class="item">
+  <h3>Skills</h3>
+  <img src="image/left2.svg" alt="">
+</li>
+<hr>
+</article>
+  </section>
+  <section class="contact">
+    <article class="contact_footer">
+      <h1>contact me</h1>
+  <img src="image/contact form shapes-mobile@2x.svg" alt="" class="mobile-img">
+      <p>If you have an application you are interested
+         in developing, a feature that you need built or a project
+          that needs coding. I’d
+         love to help with it</p>
+         <form class="form-start">
+          <input type="name" type="text" name="name" class="name" placeholder="write your name"maxlength="30">
+          <input type="email" type="email" name="email" class="email" placeholder="write your email" required>
+          <textarea type="message" type="text" name="message" class="message" placeholder="write your message" maxlength="500"></textarea>
+          <button class="start">Get in touch</button>
+         </form>
```

Comment:
- [ OPTIONAL] You did great implementing the contact form but according to the project requirement, This is not a requirement for this milestone 🧇 [PROJECT REQUIREMENT](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)